usr/src/man/man3head/tcp.h.3head

```diff
@@ -66,11 +66,24 @@ socket option at the \fBIPPROTO_TCP\fR level:
 Avoid coalescing of small segments.
 .RE
 
```

Comment:
If this is an LX-only change you should not update the illumos man page.

Reply:
The change is for OS as well not just LX